lib/stores/FileStore.ts

```diff
@@ -32,16 +34,19 @@ const log = debug('tus-node-server:stores:filestore')
 export default class FileStore extends DataStore {
   directory: string
   configstore: Store
+  expirationPeriodInMilliseconds: number
 
-  constructor({directory, configstore}: Options) {
+  constructor({directory, configstore, expirationPeriodInMilliseconds}: Options) {
     super()
     this.directory = directory
     this.configstore = configstore ?? new Configstore(`${pkg.name}-${pkg.version}`)
+    this.expirationPeriodInMilliseconds = expirationPeriodInMilliseconds ?? -1
```

Comment:
Default value in `DataStore` is `0`. Why not keep same default values?

Reply:
Both values are now set to `0`.